src/reviewer.py

```diff
@@ -23,11 +23,8 @@ def review_pull_request(self):
                 return
 
```

Comment:
Printing statement if no comments are generated
```suggestion
|
```
N/A